apps/indexer/src/api/comments/by-path/[author]/[commentId]/get.ts

```diff
@@ -130,32 +130,34 @@ async function resolveCommentPathId(
 async function resolveAuthorShortId(
   authorShortId: string | Hex | `0x${string}...${string}`,
```

Comment:
unrelated to this PR but thinking the parameter is a bit confusing, will it be better to rename it to `authorId` instead? as it accepts a wide range of types